x-pack/plugin/ilm/src/test/java/org/elasticsearch/xpack/slm/SLMSnapshotBlockingIntegTests.java

```diff
@@ -85,6 +86,7 @@ public void resetSLMSettings() throws Exception {
         return Arrays.asList(MockRepository.Plugin.class, LocalStateCompositeXPackPlugin.class, IndexLifecycle.class);
     }
 
+    @Ignore("https://github.com/elastic/elasticsearch/issues/47689")
```

Comment:
can you use `@AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/47689")` here instead?

Reply:
done, thanks